compiler/src/dmd/lexer.d

```diff
@@ -51,6 +51,7 @@ struct CompileEnv
     bool previewIn;          /// `in` means `[ref] scope const`, accepts rvalues
     bool ddocOutput;         /// collect embedded documentation comments
     bool shortenedMethods = true;   /// allow => in normal function declarations
+    bool obsolete;           /// warn on use of legacy code
```

Comment:
I assume there's a header update missing, otherwise how is gdc or ldc going to set this?

Reply:
done